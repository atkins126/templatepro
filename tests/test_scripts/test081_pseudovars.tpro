{{for cust in customers}}
{{:cust.@@index}}. {{:cust.Name}} - This index is {{if cust.@@odd }}odd{{endif}}{{if cust.@@even }}even{{endif}}
{{endfor}}