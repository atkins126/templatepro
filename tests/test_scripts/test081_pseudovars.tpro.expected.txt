1. Ford - This index is odd
2. Ferrari - This index is even
3. Lotus - This index is odd
4. FCA - This index is even
5. Hyundai - This index is odd
6. De Tomaso - This index is even
7. Dodge - This index is odd
8. Tesla - This index is even
9. Kia - This index is odd
10. Tata - This index is even
11. Volkswagen - This index is odd
12. Audi - This index is even
13. Skoda - This index is odd
